plugins/modules/idrac_user.py

```diff
@@ -59,7 +59,12 @@
         access virtual console, access virtual media, and execute debug commands.
       - A user with C(ReadOnly) privilege can only log in to iDRAC.
       - A user with C(None), no privileges assigned.
+      - Will be ignored, if custom_privilege parameter is provided.
     choices: [Administrator, ReadOnly, Operator, None]
+  custom_privilege:
+    type: int
+    description:
```

Comment:
Please add the attribute version_added to the parameter doc as below.

version_added: 8.1.0